demuxer.go

```diff
@@ -78,6 +85,13 @@ func DemuxerOptPacketsParser(p PacketsParser) func(*Demuxer) {
 	}
 }
 
+// DemuxerOptPacketFilter returns the option to set the packet filter
+func DemuxerOptPacketFilter(p PacketFilter) func(*Demuxer) {
```

Comment:
Could your rename the variable from `p` to `s`?

Reply:
Done